test/application_system_test_case.rb

```diff
@@ -0,0 +1,5 @@
+require "test_helper"
+
+class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
```

Comment:
It looks like you've chosen RSpec for your testing framework. Does it make sense to keep Minitest still in the project?